larq_compute_engine/tflite/kernels/bconv2d_params.h

```diff
@@ -25,10 +25,12 @@ typedef struct {
   std::int64_t channels_out{0};
 
   // strides
-  std::int64_t strides[4] = {};
+  std::int64_t stride_height{0};
+  std::int64_t stride_width{0};
```

Comment:
Now that we are changing this anyway, we should change it to `std::int32_t` to match the MLIR spec (as well as the dilations). And update the `.AsInt64()` flexbuffer call to `.AsInt32()`.

Originally we had this as 64-bit, I think because the flexbuffer function `AsInt64()` seemed more general than the `.AsInt32` one, and with the old non-MLIR converter we weren't sure what the actual data-type in the flexbuffer was, and `.AsInt64()` would convert correctly from any flexbuffer type.

Reply:
Good point, I changed it to `int32` in b1a0b2f61df035a9e159192c7f857bbd5ae3223b